### DISCUSSION TITLE
[限时委托]求仙舟语翻译

### DISCUSSION THREAD

**Discussion by a-132-NIDIE** [share121/inter-knot#1335] [其他]
最近计划到朱明旅游，突然发现仙舟不说通用语！
天呐，明天就要到黑塔空间站登车了
特求仙舟语翻译同行，包通票
在线等很急

委托截止日期：明日10：00
报酬：300星琼

**Comment by uidsd604** [comment]
不是哥们，哪颗星球挖出来的，现在联觉信标不是早就已经已经普及开了吗，你怎么还没有，去买一个就好了

**Reply by a-132-NIDIE** [reply]
啊？感谢感谢
[转账]300星琼

**Comment by xcshhhh** [comment]
你好，这个委托我接了！

**Comment by Inter-Knot-User** [comment]
罗浮应该有许多翻译员的，再加上现在罗浮在搞什么活动，人应该更多

**Comment by su2569** [comment]
伙计，你的联觉信标坏了？
Dude, your synesthesia beacon is broken?
![仙舟语](https://f.cdn-static.cn/81470_17233590662722.jpg)

**Comment by dengyunzhe** [comment]
这个委托我接了

**Comment by Hongxuli2006** [comment]
4楼挺厉害啊

**Comment by Jaserrr** [comment]
4楼仙舟语本当上手

**Comment by Gmadelyn7** [comment]
别担心联觉信标会自动翻译的

**Comment by WORLD-executeME** [comment]
去搞个联觉信标吧

**Comment by GanlinSama** [comment]
4F是刚过仙舟语八级吧？
